corehq/blobs/migrate.py

```diff
@@ -499,6 +500,12 @@ def migrate(self, filename=None, reset=False, max_retry=2, chunk_size=100):
         xform.SubmissionErrorLog,
         ("HQSubmission", xform.XFormInstance),
     ], CouchAttachmentMigrator),
+    Migrator("cases", [
+        cases.CommCareCase,
+        ('CommCareCase-deleted', cases.CommCareCase),
+        ('CommCareCase-Deleted', cases.CommCareCase),
+        ('CommCareCase-Deleted-Deleted', cases.CommCareCase),
```

Comment:
I'm pretty sure these are the only classes that need to be migrated. There is a [`SupplyPointCase`](https://github.com/dimagi/commcare-hq/blob/c334bd9ed7f1c8dd7a8ac8329237349a68d4b3a2/corehq/apps/commtrack/models.py#L285) subclass of `CommCareCase`, but it looks like it's never saved with that doctype (at least there were none saved in couch on prod when I looked last week). Also there are a bunch of subclasses in the `custom` code dir, but it looked like they were all reporting helpers and never saved (and there were none saved in prod couch).